src/moonlink_backend/Cargo.toml

```diff
@@ -8,6 +8,7 @@ license = { workspace = true }
 default = ["storage-fs"]
 
 profiling = ["console-subscriber"]
+test-utils = []
```

Comment:
please use moonlink one: https://github.com/Mooncake-Labs/moonlink/blob/2d8fc2b349e55e0c10072d05adf002aace8f5753/src/moonlink/Cargo.toml#L10
otherwise each crate has its own separate feature